docs/content/mongocxx-v3/installation/advanced.md

```diff
@@ -62,8 +62,8 @@ Users have the option to build `mongocxx` as a shared library that has staticall
 cmake ..                                            \
     -DCMAKE_BUILD_TYPE=Release                      \
     -DBUILD_SHARED_LIBS_WITH_STATIC_MONGOC=ON       \
-    -DCMAKE_PREFIX_PATH=/opt/mongo-c-driver         \
-    -DCMAKE_INSTALL_PREFIX=/opt/mongo-cxx-driver
+    -DCMAKE_PREFIX_PATH=$HOME/mongo-c-driver         \
+    -DCMAKE_INSTALL_PREFIX=$HOME/mongo-cxx-driver
```

Comment:
```suggestion
```
Suggest removing as this section is focused on `BUILD_SHARED_LIBS_WITH_STATIC_MONGOC`.